### ISSUE TITLE
`\` inside a table deletes the entire row from the table

### ISSUE THREAD

**Issue by treeman** [hellux/jotdown#54]
Minimal example:

```md
| `\` |
```

Produces:

```html
<table>
<tr>
</tr>
</table>
```

Can be worked around with raw inline:

```md
| `<code>\</code>`{=html} |
```

To produce the expected:

```html
<table>
<tr>
<td><code>\\</code></td>
</tr>
</table>
```

**Comment by hellux** [comment]
On Mon Jul 1, 2024 at 7:13 AM CEST, Jonas Hietala wrote:
> Minimal example:
>
> ```md
> | `\` |
> ```
>
> Produces:
>
> ```html
> <table>
> <tr>
> </tr>
> </table>
> ```
>
> Can be worked around with raw inline:
>
> ```md
> | `<code>\</code>`{=html} |
> ```
>
> To produce the expected:
>
> ```html
> <table>
> <tr>
> <td><code>\\</code></td>
> </tr>
> </table>
> ```

Thanks for the report!

It seems to be a general problem with escapes having effect in inline verbatim, e.g.

```
`foo\` bar
```

produces

```
<p><code>foo\` bar</code></p>
```

because the backtick is escaped, which it should not according to the
djot syntax reference.

**Comment by hellux** [comment]
fixed as of release 0.4.1

**Comment by treeman** [comment]
That was fast, thanks!
